frontend/src/Components/Feedback.jsx

```diff
@@ -3,19 +3,19 @@ import React from 'react'
 export default (props) => {
```

Comment:
anonymous functional component? this article from 2017 says functionals should always have a name or things might not work as well and its not semantically right. 
[do-not-use-anonymous-functions-to-construct-react-functional-components](https://medium.com/@stevemao/do-not-use-anonymous-functions-to-construct-react-functional-components-c5408ec8f4c7)

there's also this one from 2018:
[react-devtools-and-unknown-components](https://dev.to/fargrim/react-devtools-and-unknown-components-3ni6)

couldnt find anything more recent so you might know more than i do

Reply:
I did't know about that and I don't know if improvements have been done since these articles, I've seen some tutorials using anonymous functions but to be safe, I rather changing that

Reply:
named